--- origsrc/renderproto-0.9.3/Makefile.am	2007-08-31 16:18:26.000000000 -0500
+++ src/renderproto-0.9.3/Makefile.am	2007-09-25 22:01:33.562500000 -0500
@@ -6,7 +6,6 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = renderproto.pc
 
-renderdocdir = $(datadir)/doc/$(PACKAGE)
-renderdoc_DATA = renderproto.txt
+doc_DATA = renderproto.txt
 
 EXTRA_DIST = autogen.sh renderproto.pc.in $(renderdoc_DATA)
